Source/AliveLibAO/Abe.cpp

```diff
@@ -5371,7 +5395,20 @@ void Abe::State_138_ElumUnmountEnd_42E390()
 
 void Abe::State_139_ElumMountBegin_42E090()
 {
-    NOT_IMPLEMENTED();
+    if (!field_104_pending_resource_count && field_10_anim.field_4_flags.Get(AnimFlags::eBit18_IsLastFrame))
+    {
+        while (!ResourceManager::FreeResource_455550(field_1A4_resources.res[61]))
+        {
+            // Empty
+        }
+
+        field_1A4_resources.res[61] = {};
```

Comment:
might be better to use = nullptr; if its only clearing 1 element rather than all of them